Library/Homebrew/extend/ARGV.rb

```diff
@@ -125,7 +125,7 @@ def git?
   end
 
   def homebrew_developer?
-    include?("--homebrew-developer") || !ENV["HOMEBREW_DEVELOPER"].nil?
+    !ENV["HOMEBREW_DEVELOPER"].nil?
```

Comment:
Isn’t it strange to have a method defined on `ARGV` that now have nothing to do with `ARGV`? :wink: